server/src/app.ts

```diff
@@ -43,6 +55,48 @@ app.get('/visit', (_, res) => {
   res.sendFile(path.join(__dirname, 'public/visit.html'));
 });
 
+/**
+ * route: /connectToRoom
+ * purpose: Calling: connect to an existing room
+ */
+app.use('/api/connectRoomsCall', cors(), connectRoomsCall);
+/**
+ * route: /startTranscription
+ * purpose: Start transcription for an established call
+ */
+app.use('/api/startTranscription', cors(), startTranscription);
+
+/**
+ * route: /stopTranscription
+ * purpose: Stop transcription for an established call
+ */
+app.use('/api/stopTranscription', cors(), stopTranscriptionForCall);
+
+/**
+ * route: /fetchTranscript
+ * purpose: Fetch an existing transcription
+ */
+app.use('/api/fetchTranscript', cors(), fetchTranscript);
+
+/**
+ * route: /startCallWithTranscription
+ * purpose: Start a new group call with transcription
+ */
+app.use('/api/startCallWithTranscription', cors(), startCallWithTranscription);
+/**
+ * route: /callAutomationEvent
+ * purpose: Call Automation: receive call automation events
+ */
+app.use('/api/callAutomationEvent', cors(), callAutomationEvent);
+
+/**
+ * route: /summarizeTranscript
+ * purpose: Sends transcript to AI summarization service
+ */
+app.use('/api/summarizeTranscript', cors(), summarizeTranscript);
+
+app.use('/api/events', cors(), events);
```

Comment:
`/events` is very generic, there is also `/callAutomationEvent`, rename this to be more specific on its purpose